### ISSUE TITLE
Player gets error

### ISSUE THREAD

**Issue by Ikabodo** [Handyfon/Farchievements#6]
![image](https://user-images.githubusercontent.com/7766890/126013505-d4cc54fd-e395-41b3-ab6b-3992c95fe70a.png)
One of my player's is getting this strange error. He cannot see any achievements.

I tried logging in to the player's account and I can se them, but he cannot. Tried on two different computers.

I'll try to find more info, but for now it's kinda puzzling.

Chrome

**Comment by xlilcasper** [comment]
Sent this on discord but posting it here just in case.

basically it has to do with the fact that the setting clientdata can be empty in a new world. I added a line to AchievementsScreen.js at line 308
**if(mysettings == "") mysettings = ":"**

This allowed syncing to happen if mysettings was empty. I also changed line 73 of AchievementsScreen.html (not sure if needed). Changed it to

if(**game.settings.get('farchievements', 'clientdata').toString().length > 0 && (** game.settings.get('farchievements', 'clientdata').toString().split(":")[1].includes("," + i + ",") || game.settings.get('farc...

that way if clientdata is 0 length, it heads to the else and displays it properly
